docs/deployment_patterns/how_to_instantiate_a_data_context_hosted_environments.md

```diff
@@ -25,18 +25,22 @@ The following guide gives an overview of creating an in-code Data Context includ
 
 The following guides will contain examples for each environment we have tested out:
 
-- [AWS EMR](./how_to_instantiate_a_data_context_on_an_emr_spark_cluster)
-- [Databricks](./how_to_instantiate_a_data_context_on_databricks_spark_cluster)
-- :ref:`deployment_google_cloud_composer`
-- :ref:`deployment_astronomer`
+- [How to instantiate a Data Context on an EMR Spark cluster](./how_to_instantiate_a_data_context_on_an_emr_spark_cluster)
+- [How to instantiate a Data Context on Databricks Spark cluster](./how_to_instantiate_a_data_context_on_databricks_spark_cluster)
 
 
 Step 2: Create Expectation Suites and add Expectations
 -------------------------------------------------------
 
-If you want to create an Expectation Suite in your environment without using the CLI, you can simply follow this guide: [TODO: Add content and reference to "How to Create a New Expectation Suite Without the CLI"].
+Create your Expectation Suites and save the state to a validator.
```

Comment:
This will need to be updated once we create the page "How to Create a New Expectation Suite Without the CLI".

Reply:
I'm wondering if there is a way to use existing documentation to link in the interim, since this content would be pretty much the same (e.g. https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/database/postgres/)

Reply:
This link is great! I added a little color to it as well to get them across the finish line.